<?xml version="1.0" encoding="UTF-8"?>
<class xmlns="http://xml.phpdox.net/src" full="Symfony\Component\Debug\Exception\DummyException" namespace="Symfony\Component\Debug\Exception" name="DummyException" abstract="false" final="false" start="21" end="23">
  <file path="/home/web/example/src/freegame_developer/vendor/symfony/debug/Exception" file="DummyException.php" realpath="/home/web/example/src/freegame_developer/vendor/symfony/debug/Exception/DummyException.php" size="591" time="2016-09-06T07:26:07+00:00" unixtime="1473146767" sha1="7c6248aeba9703a38ad907902b6c907816dae6d0" relative="freegame_developer/vendor/symfony/debug/Exception/DummyException.php"/>
  <docblock>
    <description/>
    <author value="Fabien Potencier &lt;fabien@symfony.com&gt;"/>
    <deprecated value="since version 2.5, to be removed in 3.0."/>
  </docblock>
  <extends full="ErrorException" namespace="" name="ErrorException"/>
</class>
